test_collections/test_tower_organization.yml

```diff
@@ -0,0 +1,19 @@
+---
+- hosts: localhost
+  gather_facts: false
+  vars:
+    collection_id: awx.awx
+    src_file: test_tower_organization.j2
+    dest_file: tower_collection_actual.yml
+  tasks:
+    - name: "Create playbook to call with collection"
+      template:
+        src: "{{ src_file }}"
+        dest: "{{ dest_file }}"
+
+    - include: "{{ dest_file }}"
+
+    - name: "Delete dynamic playbook"
+      file:
+        path: "{{ dest_file }}"
+        state: absent
```

Comment:
I don't see why this _file_ is necessary, because you could just use the `tower_collection_smoke.yml` playbook with extra-vars.

Reply:
yes, i got it, i only created that to test and to understand how it works. I will remove.